pkgs/development/python-modules/pywebcopy/default.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  pytestCheckHook,
+  setuptools,
+  cachecontrol,
+  lxml-html-clean,
+  requests,
+  six,
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

```suggestion
buildPythonPackage rec {
```

that should also come from callPackage arguments